content/scripts/print-diff.js

```diff
@@ -0,0 +1,27 @@
+const chalk = require("chalk");
+
+function printBasicDiff(str1, str2) {
+  const oldLines = str1.split("\n");
+  const newLines = str2.split("\n");
```

Comment:
I don't think so. If the string is `One\r\nTwo\r\n` it will just split it as `['One\r', 'Two\r']` and it'll work the same. 
Seems to work when I created a test case like this:
```
▶ python
Python 3.8.1 (default, Feb 18 2020, 21:11:26)
[Clang 11.0.0 (clang-1100.0.33.17)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> with open('/tmp/text1.txt', 'w') as f:
...   f.write('One\r\nTwo\r\n')
...
10
>>> with open('/tmp/text2.txt', 'w') as f:
...   f.write('One\r\nThree\r\n')
...
12
```

Reply:
However, I did find a bug in the function anyway. If the number of lines in the text is less than the padding it prints after the affected line, it would have printed `undefined`. Fixed that with an if statement.